styles.css

```diff
@@ -167,21 +250,70 @@
 }
 
 .work-section {
-  padding-top: 104px;
-  padding-bottom: 104px;
+  box-sizing: border-box;
+  background-color: #f1f5f1;
+  justify-content: center;
   display: grid;
-  grid-template-columns: 327px;
+  grid-template-columns: 320px;
   grid-template-rows: repeat(4, auto);
   row-gap: 104px;
-  background-color: #f1f5f1;
-  justify-content: center;
+  padding-bottom: 104px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
None of your project cards match the template design. Please refer to the template design and fix them.